packages/undo-manager/src/sync-undo-manager.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * WordPress dependencies
+ */
+import { getSyncProvider } from '@wordpress/sync';
+
+/**
+ * Internal dependencies
+ */
+import type { UndoManager, HistoryRecord } from './types';
+
+/**
+ * Creates the sync powered undo manager.
+ *
+ * @return {UndoManager} Undo manager.
+ */
+export function createSyncUndoManager() {
+	return {
+		/**
+		 * Record changes into the history.
+		 *
+		 * @param {HistoryRecord=} record   A record of changes to record.
+		 * @param {boolean}        isStaged Whether to immediately create an undo point or not.
```

Comment:
I believe `=` means optional?
```suggestion
		 * @param {HistoryRecord} record    A record of changes to record.
		 * @param {boolean=}       isStaged Whether to immediately create an undo point or not.
```

Reply:
That was me accidentally doing a typo, sorry. It's not supposed to be there like that